src/checkbox/CheckBox.js

```diff
@@ -79,16 +79,6 @@ styles = StyleSheet.create({
     flexDirection: 'row',
     alignItems: 'center'
   },
-  container: {
```

Comment:
@jenessawhite What's the reason for this change? This seems to be useful for styling the component in line 22